apps/ai_reports/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+from adhocracy4.comments.models import Comment
+from adhocracy4.models.base import TimeStampedModel
+
+
+class AiReport(TimeStampedModel):
+    is_pending = models.BooleanField(default=True)
+    category = models.CharField(max_length=50)
+    explanation = models.TextField()
```

Comment:
can you rename this field as ai_feedback to be inline with the moderator_feedback:
  `  moderator_feedback = ModeratorCommentFeedbackSerializer(read_only=True)`
  https://github.com/liqd/adhocracy-plus/blob/1b6c3b73fba968c856322c191d4ef345b989ee55/apps/moderatorfeedback/serializers.py#L36

Reply:
I thought "explanation" was quite fitting but I am also happy to change it. What do you think @goapunk @philli-m ?

Reply:
Maybe we can talk about it in standup, I do hate the word `ai` but it also is more consistent with the naming we have.

Reply:
Also this field will be added to the serialiser file am pointing above bcz we need it for the comment section, to appear below the moderator feedback on the website, so it will make teh code more readable.

Reply:
resolved: we decided to leave it as is for now and change if we need later.